cmd/influxd/run.go

```diff
@@ -121,6 +122,26 @@ func (cmd *RunCommand) Run(args ...string) error {
 	return nil
 }
 
+// CheckConfig validates the configuration
+func (cmd *RunCommand) CheckConfing() {
+	if !(cmd.config.Data.Enabled || cmd.config.Broker.Enabled) {
+		cmd.Logger.Fatal("Broker and/or data options must be enabled.  Run `influxd config` for valid config options.")
```

Comment:
This is purely a style issue, but I would phrase this differently.

`Node must be configured as a broker node, data node, or as both. Run 'influxd config' to generate a valid configuration.'